Jenkinsfile_CNP

```diff
@@ -71,7 +71,7 @@ env.BEFTA_S2S_CLIENT_ID = "ccd_gw"
 env.OAUTH2_CLIENT_ID = "ccd_gateway"
 env.OAUTH2_REDIRECT_URI = "https://www-ccd.aat.platform.hmcts.net/oauth2redirect"
 env.DM_STORE_BASE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
-env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-pr-554.service.core-compute-preview.internal"
+env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-pr-552.service.core-compute-preview.internal"
```

Comment:
should this be pointing to a PR in master?

Reply:
pointing now to aat